<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="quit_game">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
qproperty-icon: url(:/qss_icons/light/rc/window_close_focus.png)
}</string>
              </property>
              <property name="text">
               <string>Quit</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="rerun_game_2">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
qproperty-icon: 
url(:/qss_icons/light/rc/arrow_left_focus.png)
}</string>
              </property>
              <property name="text">
               <string>Rerun Game</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pause">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
qproperty-icon: 
url(:/qss_icons/light/rc/toolbar_move_horizontal_focus.png)
}</string>
              </property>
              <property name="text">
               <string>Pause Game</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="zoom_label">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Zoom</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10000</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <item row="4" column="0">
           <widget class="QLabel" name="level_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}
</string>
            </property>
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="enemies_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}
</string>
            </property>
            <property name="text">
             <string>Enemies</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="time_label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}
</string>
            </property>
            <property name="text">
             <string>Elapsed Time</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="lcdTime">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLCDNumber{
    color: rgb(220, 50, 47);    
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLCDNumber" name="lcdLevel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLCDNumber{
    color: rgb(220, 50, 47);    
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="health_packs">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}
</string>
            </property>
            <property name="text">
             <string>Health Packs</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLCDNumber" name="lcdEnemies">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLCDNumber{
    color: rgb(220, 50, 47);    
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLCDNumber" name="lcdHealth">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLCDNumber{
    color: rgb(220, 50, 47);    
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="health_label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Health</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="health">
            <property name="whatsThis">
             <string/>
            </property>
            <property name="accessibleName">
             <string>Energy </string>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"> QProgressBar::chunk {
     background-color: rgb(51, 209, 122);
 }
</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="energy_label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Energy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="energy">
            <property name="whatsThis">
             <string/>
            </property>
            <property name="accessibleName">
             <string>Energy </string>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"> QProgressBar::chunk {
     background-color: rgb(246, 211, 45);
 }
</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X coordinate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y coordinate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="x_path">
            <property name="text">
             <string>-1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="y_path">
            <property name="text">
             <string>-1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>399</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="midLineWidth">
           <number>40</number>
          </property>
          <property name="text">
           <string>If non-valid coordinates, path routes automatically
 to the doorway (next level)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="path_find_trigger">
          <property name="text">
           <string>Find Shortest Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="type_command">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Type command</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textEdit">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="text_mode"/>
    <addaction name="sprite_mode"/>
    <addaction name="colour_mode"/>
   </widget>
   <addaction name="view_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="end_game">
   <property name="text">
    <string>End Game</string>
   </property>
  </action>
  <action name="start_game">
   <property name="text">
    <string>Start Game</string>
   </property>
  </action>
  <action name="rerun_game">
   <property name="text">
    <string>Re-run Game</string>
   </property>
  </action>
  <action name="manual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="automatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="save_game">
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="text_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="sprite_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sprite</string>
   </property>
  </action>
  <action name="colour_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colours</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
